csrankings-a.csv

```diff
@@ -2503,3 +2503,6 @@ Aïda Ouangraoua,Université de Sherbrooke,https://www.usherbrooke.ca/informatiq
 Ángel Viña Castiñeiras,University of A Coruña,https://pdi.udc.es/es/File/Pdi/FK7AF,NOSCHOLARPAGE
 Ángeles Saavedra,University of A Coruña,https://lbd.udc.es/ShowResearcherInformation.do?id=5,IYkBYzQAAAAJ
 Ángeles Saavedra Places,University of A Coruña,https://lbd.udc.es/ShowResearcherInformation.do?id=5,IYkBYzQAAAAJ
+Anca L. Ralescu,University of Cincinnati,https://researchdirectory.uc.edu/p/ralescal?_gl=1*1foalj6*_ga*NzIyNzk3MTI1LjE2NzE0NTExNDc.*_ga_7H1FCKXZ4Y*MTcxMDg2MDgwNS4yNjUuMS4xNzEwODgzMzI2LjYwLjAuMA,NOSCHOLARPAGE

```

Comment:
1. All of these should be placed in alphabetical order in their respective file.
2. The analytics stuff (after the ?) should be omitted in all home pages.

Reply:
3. Research faculty (e.g. https://researchdirectory.uc.edu/p/jeggaa) are not eligible for inclusion and must be removed.